AudioManagerUtilities/include/CAmSocketHandler.h

```diff
@@ -305,14 +305,13 @@ class CAmSocketHandler
     internal_codes_t mInternalCodes;
     sh_pollHandle_t mSignalFdHandle;
     VectorListPoll_t mListActivePolls;
+    const std::thread::id mThreadID; //!< Socket handler thread id used to check if the calls come from the same thread
```

Comment:
The description of the commit does not explain that this functionality was added to check the threads this way.  (It also does not have a short first summary line, as a git-commit should have)